zerver/lib/zulip_update_announcements.py

```diff
@@ -0,0 +1,206 @@
+from dataclasses import dataclass
+from datetime import timedelta
+from typing import List, Optional
+
+from django.conf import settings
+from django.db import transaction
+from django.db.models import Q, QuerySet
+from django.utils.timezone import now as timezone_now
+from django.utils.translation import override as override_language
+
+from zerver.actions.message_send import (
+    do_send_messages,
+    internal_prep_huddle_message,
+    internal_prep_stream_message,
+)
+from zerver.lib.message import SendMessageRequest
+from zerver.models.realm_audit_logs import RealmAuditLog
+from zerver.models.realms import Realm
+from zerver.models.users import UserProfile, get_system_bot
+
+
+@dataclass
+class ZulipUpdateAnnouncement:
+    level: int
+    message: str
+
+
+# We don't translate the announcement message because they are quite unlikely to be
+# translated during the time between when we draft them and when they are published.
+zulip_update_announcements: List[ZulipUpdateAnnouncement] = [
+    ZulipUpdateAnnouncement(
+        level=1,
+        message=(
+            "This topic will receive automated updates about new Zulip features.\n"
+            "\n"
+            "If you like, you can [configure]({settings_url}) a different stream "
```

Comment:
I'm thinking this should be tweaked, so that it makes sense for both admins and non-admins. Let's try changing "If you like, you can" -> "Organization administrators can".